bot/pagination.py

```diff
@@ -129,26 +141,31 @@ def event_check(reaction_: Reaction, user_: Member):
 
             no_restrictions = (
                 # Pagination is not restricted
-                not restrict_to_user or
+                not restrict_to_user
+                or
                 # The reaction was by a whitelisted user
```

Comment:
naw, this part doesn't work for me. it'd be better to have an inline comment, I think. or something else. just find any other solution. we can't have that lonely `or`